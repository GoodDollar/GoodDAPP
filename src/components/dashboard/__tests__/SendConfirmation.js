import React from 'react'
// Note: test renderer must be required after react-native.
import renderer from 'react-test-renderer'
import { getWebRouterComponentWithMocks } from './__util__'

describe('SendConfirmation', () => {
  it('renders without errors', () => {
    const SendConfirmation = getWebRouterComponentWithMocks('../SendConfirmation')
    const tree = renderer.create(<SendConfirmation />)
    expect(tree.toJSON()).toBeTruthy()
  })

  it('matches snapshot', () => {
    const SendConfirmation = getWebRouterComponentWithMocks('../SendConfirmation')
    const component = renderer.create(<SendConfirmation />)
    const tree = component.toJSON()
    expect(tree).toMatchSnapshot()
  })
})
