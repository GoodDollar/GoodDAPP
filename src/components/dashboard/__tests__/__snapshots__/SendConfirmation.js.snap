// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SendConfirmation matches snapshot 1`] = `
<div
  className="css-view-1dbjc4n"
  style={
    Object {
      "WebkitBoxFlex": 1,
      "WebkitFlexBasis": "0%",
      "WebkitFlexGrow": 1,
      "WebkitFlexShrink": 1,
      "flexBasis": "0%",
      "flexGrow": 1,
      "flexShrink": 1,
      "msFlexNegative": 1,
      "msFlexPositive": 1,
      "msFlexPreferredSize": "0%",
    }
  }
>
  <div
    className="css-view-1dbjc4n"
    data-name="viewWrapper"
    style={
      Object {
        "WebkitBoxDirection": "normal",
        "WebkitBoxFlex": 1,
        "WebkitBoxOrient": "vertical",
        "WebkitFlexDirection": "column",
        "WebkitFlexGrow": 1,
        "backgroundImage": "linear-gradient(to bottom, #00AFFF, #2DC0F7, #28C0EF, #23C0E7, #1EC1DF, #19C1D7, #14C1CF, #0FC2C7, #0FC2C7, #0AC2BF, #05C2B7, #00C3AF)",
        "display": "flex",
        "flexDirection": "column",
        "flexGrow": 1,
        "maxHeight": "844px",
        "msFlexDirection": "column",
        "msFlexPositive": 1,
        "paddingBottom": "8px",
        "paddingLeft": "8px",
        "paddingRight": "8px",
        "paddingTop": "8px",
        "position": "relative",
        "width": "100%",
      }
    }
  >
    <div
      className="css-view-1dbjc4n"
      style={
        Object {
          "WebkitBoxPack": "center",
          "WebkitJustifyContent": "center",
          "backgroundColor": "rgba(255,255,255,1.00)",
          "borderBottomLeftRadius": "5px",
          "borderBottomRightRadius": "5px",
          "borderTopLeftRadius": "5px",
          "borderTopRightRadius": "5px",
          "height": "62px",
          "justifyContent": "center",
          "marginBottom": "8px",
          "msFlexPack": "center",
          "paddingBottom": "8px",
          "paddingLeft": "12px",
          "paddingRight": "8px",
          "paddingTop": "8px",
        }
      }
    >
      <div
        className="css-view-1dbjc4n"
        style={
          Object {
            "WebkitAlignItems": "center",
            "WebkitBoxAlign": "center",
            "WebkitBoxDirection": "reverse",
            "WebkitBoxOrient": "horizontal",
            "WebkitBoxPack": "justify",
            "WebkitFlexDirection": "row-reverse",
            "WebkitJustifyContent": "space-between",
            "alignItems": "center",
            "flexDirection": "row-reverse",
            "justifyContent": "space-between",
            "msFlexAlign": "center",
            "msFlexDirection": "row-reverse",
            "msFlexPack": "justify",
          }
        }
      >
        <div
          aria-disabled={true}
          className="css-view-1dbjc4n"
          disabled={true}
          onKeyDown={[Function]}
          onKeyUp={[Function]}
          onResponderGrant={[Function]}
          onResponderMove={[Function]}
          onResponderRelease={[Function]}
          onResponderTerminate={[Function]}
          onResponderTerminationRequest={[Function]}
          onStartShouldSetResponder={[Function]}
          style={
            Object {
              "MozTransitionProperty": "opacity",
              "MozUserSelect": "none",
              "WebkitTransitionDuration": "0.15s",
              "WebkitTransitionProperty": "opacity",
              "WebkitUserSelect": "none",
              "backgroundColor": "rgba(203,203,203,1.00)",
              "borderBottomColor": "rgba(163,163,163,1.00)",
              "borderBottomLeftRadius": "21px",
              "borderBottomRightRadius": "21px",
              "borderBottomWidth": "1px",
              "borderLeftColor": "rgba(163,163,163,1.00)",
              "borderLeftWidth": "1px",
              "borderRightColor": "rgba(163,163,163,1.00)",
              "borderRightWidth": "1px",
              "borderTopColor": "rgba(163,163,163,1.00)",
              "borderTopLeftRadius": "21px",
              "borderTopRightRadius": "21px",
              "borderTopWidth": "1px",
              "height": "42px",
              "msUserSelect": "none",
              "transitionDuration": "0.15s",
              "transitionProperty": "opacity",
              "userSelect": "none",
              "width": "42px",
            }
          }
        >
          <div
            className="css-view-1dbjc4n"
            style={
              Object {
                "backgroundColor": "rgba(0,0,0,0.00)",
                "borderBottomLeftRadius": "20px",
                "borderBottomRightRadius": "20px",
                "borderTopLeftRadius": "20px",
                "borderTopRightRadius": "20px",
                "height": "40px",
                "width": "40px",
              }
            }
          >
            <div
              className="css-view-1dbjc4n"
              style={
                Object {
                  "WebkitFlexBasis": "auto",
                  "borderBottomLeftRadius": "20px",
                  "borderBottomRightRadius": "20px",
                  "borderTopLeftRadius": "20px",
                  "borderTopRightRadius": "20px",
                  "flexBasis": "auto",
                  "height": "40px",
                  "msFlexPreferredSize": "auto",
                  "overflowX": "hidden",
                  "overflowY": "hidden",
                  "width": "40px",
                  "zIndex": 0,
                }
              }
            >
              <div
                className="css-view-1dbjc4n"
                style={
                  Object {
                    "backgroundColor": "rgba(0,0,0,0.00)",
                    "backgroundImage": "url(\\"unknownProfile.svg\\")",
                    "backgroundPosition": "center",
                    "backgroundRepeat": "no-repeat",
                    "backgroundSize": "cover",
                    "bottom": "0px",
                    "height": "100%",
                    "left": "0px",
                    "position": "absolute",
                    "right": "0px",
                    "top": "0px",
                    "width": "100%",
                    "zIndex": -1,
                  }
                }
              />
              <img
                alt=""
                className="css-accessibilityImage-9pa8cd"
                draggable={false}
                src="unknownProfile.svg"
              />
            </div>
          </div>
        </div>
      </div>
    </div>
    <div
      className="css-view-1dbjc4n"
      style={
        Object {
          "WebkitBoxDirection": "normal",
          "WebkitBoxFlex": 1,
          "WebkitBoxOrient": "vertical",
          "WebkitBoxPack": "justify",
          "WebkitFlexDirection": "column",
          "WebkitFlexGrow": 1,
          "WebkitJustifyContent": "space-between",
          "backgroundColor": "rgba(255,255,255,1.00)",
          "borderBottomLeftRadius": "5px",
          "borderBottomRightRadius": "5px",
          "borderTopLeftRadius": "5px",
          "borderTopRightRadius": "5px",
          "display": "flex",
          "flexDirection": "column",
          "flexGrow": 1,
          "justifyContent": "space-between",
          "msFlexDirection": "column",
          "msFlexPack": "justify",
          "msFlexPositive": 1,
          "paddingBottom": "16px",
          "paddingLeft": "12px",
          "paddingRight": "12px",
          "paddingTop": "16px",
        }
      }
    >
      <div
        className="css-view-1dbjc4n"
        style={
          Object {
            "WebkitAlignItems": "stretch",
            "WebkitBoxAlign": "stretch",
            "WebkitBoxDirection": "normal",
            "WebkitBoxFlex": 1,
            "WebkitBoxOrient": "vertical",
            "WebkitBoxPack": "center",
            "WebkitFlexDirection": "column",
            "WebkitFlexGrow": 1,
            "WebkitJustifyContent": "center",
            "alignItems": "stretch",
            "flexDirection": "column",
            "flexGrow": 1,
            "justifyContent": "center",
            "msFlexAlign": "stretch",
            "msFlexDirection": "column",
            "msFlexPack": "center",
            "msFlexPositive": 1,
          }
        }
      >
        <div
          className="css-text-901oao"
          dir="auto"
          style={
            Object {
              "WebkitAlignItems": "center",
              "WebkitBoxAlign": "center",
              "WebkitBoxPack": "center",
              "WebkitJustifyContent": "center",
              "alignItems": "center",
              "color": "rgba(66,69,74,1.00)",
              "direction": "ltr",
              "display": "flex",
              "fontFamily": "Roboto",
              "fontSize": "22px",
              "fontWeight": "500",
              "justifyContent": "center",
              "lineHeight": "30px",
              "msFlexAlign": "center",
              "msFlexPack": "center",
              "textAlign": "center",
              "textDecoration": "none",
              "textTransform": "none",
            }
          }
        >
          To complete the transaction
copy the link and share it
with your recipient.
        </div>
        <div
          className="css-view-1dbjc4n"
          style={
            Object {
              "WebkitAlignItems": "stretch",
              "WebkitBoxAlign": "stretch",
              "WebkitBoxDirection": "normal",
              "WebkitBoxOrient": "vertical",
              "WebkitBoxPack": "justify",
              "WebkitFlexDirection": "column",
              "WebkitJustifyContent": "space-between",
              "alignItems": "stretch",
              "flexDirection": "column",
              "justifyContent": "space-between",
              "marginBottom": "50px",
              "marginTop": "50px",
              "msFlexAlign": "stretch",
              "msFlexDirection": "column",
              "msFlexPack": "justify",
            }
          }
        >
          <div
            className="css-view-1dbjc4n"
            style={
              Object {
                "WebkitAlignItems": "center",
                "WebkitBoxAlign": "center",
                "WebkitBoxDirection": "normal",
                "WebkitBoxOrient": "horizontal",
                "WebkitBoxPack": "center",
                "WebkitFlexDirection": "row",
                "WebkitJustifyContent": "center",
                "alignItems": "center",
                "flexDirection": "row",
                "justifyContent": "center",
                "msFlexAlign": "center",
                "msFlexDirection": "row",
                "msFlexPack": "center",
              }
            }
          >
            <div
              className="css-view-1dbjc4n"
              style={
                Object {
                  "WebkitAlignItems": "center",
                  "WebkitBoxAlign": "center",
                  "WebkitBoxPack": "center",
                  "WebkitJustifyContent": "center",
                  "alignItems": "center",
                  "backgroundColor": "rgba(250,108,119,1.00)",
                  "borderBottomLeftRadius": "50%",
                  "borderBottomRightRadius": "50%",
                  "borderTopLeftRadius": "50%",
                  "borderTopRightRadius": "50%",
                  "display": "flex",
                  "height": "75px",
                  "justifyContent": "center",
                  "marginBottom": "24px",
                  "marginTop": "15px",
                  "msFlexAlign": "center",
                  "msFlexPack": "center",
                  "position": "relative",
                  "width": "75px",
                }
              }
            >
              <div
                className="css-text-901oao"
                dir="auto"
                style={
                  Object {
                    "color": "rgba(255,255,255,1.00)",
                    "fontFamily": "gooddollar",
                    "fontSize": "45px",
                    "fontStyle": "normal",
                    "fontWeight": "normal",
                  }
                }
              >
                
              </div>
            </div>
          </div>
          <div
            className="css-text-901oao"
            dir="auto"
            style={
              Object {
                "WebkitBoxPack": "center",
                "WebkitJustifyContent": "center",
                "color": "rgba(66,69,74,1.00)",
                "direction": "ltr",
                "display": "flex",
                "fontFamily": "Roboto",
                "fontSize": "24px",
                "fontWeight": "500",
                "justifyContent": "center",
                "lineHeight": "30px",
                "marginTop": "10px",
                "msFlexPack": "center",
                "textAlign": "center",
                "textDecoration": "none",
                "textTransform": "uppercase",
              }
            }
          >
            <div
              className="css-view-1dbjc4n"
              style={
                Object {
                  "WebkitAlignItems": "baseline",
                  "WebkitBoxAlign": "baseline",
                  "WebkitBoxDirection": "normal",
                  "WebkitBoxOrient": "horizontal",
                  "WebkitFlexDirection": "row",
                  "alignItems": "baseline",
                  "display": "flex",
                  "flexDirection": "row",
                  "msFlexAlign": "baseline",
                  "msFlexDirection": "row",
                }
              }
            >
              <span
                className="css-text-901oao css-textHasAncestor-16my406"
                dir="auto"
                style={
                  Object {
                    "color": "rgba(250,108,119,1.00)",
                    "direction": "ltr",
                    "fontFamily": "Roboto Slab",
                    "fontSize": "36px",
                    "fontWeight": "700",
                    "lineHeight": "24px",
                    "marginRight": "2px",
                    "textAlign": "right",
                    "textDecoration": "none",
                    "textTransform": "none",
                  }
                }
              >
                -.--
              </span>
              <div
                className="css-view-1dbjc4n"
                style={
                  Object {
                    "WebkitAlignItems": "center",
                    "WebkitBoxAlign": "center",
                    "WebkitBoxDirection": "normal",
                    "WebkitBoxOrient": "horizontal",
                    "WebkitBoxPack": "justify",
                    "WebkitFlexDirection": "row",
                    "WebkitJustifyContent": "space-between",
                    "alignItems": "center",
                    "display": "inline-flex",
                    "flexDirection": "row",
                    "justifyContent": "space-between",
                    "msFlexAlign": "center",
                    "msFlexDirection": "row",
                    "msFlexPack": "justify",
                  }
                }
              >
                <span
                  className="css-text-901oao css-textHasAncestor-16my406"
                  dir="auto"
                  style={
                    Object {
                      "color": "rgba(250,108,119,1.00)",
                      "direction": "ltr",
                      "fontFamily": "Roboto Slab",
                      "fontSize": "14px",
                      "fontWeight": "700",
                      "lineHeight": "22px",
                      "textAlign": "right",
                      "textDecoration": "none",
                      "textTransform": "none",
                    }
                  }
                >
                  G
                </span>
                <span
                  className="css-text-901oao css-textHasAncestor-16my406"
                  dir="auto"
                  style={
                    Object {
                      "color": "rgba(250,108,119,1.00)",
                      "direction": "ltr",
                      "fontFamily": "Roboto Slab",
                      "fontSize": "14px",
                      "fontWeight": "700",
                      "lineHeight": "22px",
                      "textAlign": "right",
                      "textDecoration": "none",
                      "textTransform": "none",
                    }
                  }
                >
                  $
                </span>
              </div>
            </div>
          </div>
        </div>
      </div>
      <div
        className="css-view-1dbjc4n"
        data-focusable={true}
        disabled={false}
        onKeyDown={[Function]}
        onKeyUp={[Function]}
        onResponderGrant={[Function]}
        onResponderMove={[Function]}
        onResponderRelease={[Function]}
        onResponderTerminate={[Function]}
        onResponderTerminationRequest={[Function]}
        onStartShouldSetResponder={[Function]}
        style={
          Object {
            "MozTransitionProperty": "opacity",
            "MozUserSelect": "none",
            "WebkitTransitionDuration": "0.15s",
            "WebkitTransitionProperty": "opacity",
            "WebkitUserSelect": "none",
            "cursor": "pointer",
            "msTouchAction": "manipulation",
            "msUserSelect": "none",
            "touchAction": "manipulation",
            "transitionDuration": "0.15s",
            "transitionProperty": "opacity",
            "userSelect": "none",
            "width": "100%",
          }
        }
        tabIndex="0"
      >
        <div
          className="css-view-1dbjc4n"
          style={
            Object {
              "width": "100%",
            }
          }
        />
      </div>
    </div>
  </div>
</div>
`;
